### PR TITLE
Patch CSS to tidy rendering and enable xdai balance

### PR DESCRIPTION
**Pull request by JamesSmartCell**
Patch CSS to tidy rendering, fix input type and enable xdai balance.

### CODE REVIEW

File: examples/action/xdai-bridge.en.shtml [hunk 1]
```diff
@@ -6,20 +6,26 @@
...
-    //if (currentTokenInstance.balance != null) {
-      //var correctedBalance = currentTokenInstance.balance / 1e+18;
-      //document.getElementById('daiVal').innerHTML = correctedBalance;
-    //}
-  }
-</script>
+	web3.tokens.dataChanged = (oldTokens, updatedTokens) => {
+		const currentTokenInstance = web3.tokens.data.currentInstance
+		if (currentTokenInstance.balance != null) {
+			var correctedBalance = currentTokenInstance.balance / 1e+18;
```
Comment (James-Sangalli) [inline examples/action/xdai-bridge.en.shtml]: should be 'xdaiVal'

File: examples/action/xdai-bridge.en.shtml [hunk 2]
```diff
@@ -6,20 +6,26 @@
...
+					height="20" alt="Dai Balance" />
+				<div id="daiVal"></div>
+			</td>
 		</tr>
 	</tbody>
 </table>
 <h2>Convert to DAI</h2>
 <form action="/">
-	<p><input type="text" id="amount"/><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAoCAYAAACfKfiZAAAABGdBTUEAALGPC/xhBQAAAcNJREFUWAntl71KxEAQx03OyuJSiIKtCIIfT+ATiOnEsxMld6X2Fgo2PoBgdXfJNT5BtNM7BLGysvQZvM5CuFzifwJzjHf5uLBbbiDszO7M/H87bGBjLZQ8vu9vxXEcWpb14HnedUl4utzpdB6TJDmQsch/ajabrpwj256ekH6v19sej8cDFFsHxBUK38r1AnszYy1rLh+AxEejUR+FVrkYQC4rQHBa4ZjZAYjsRFE0kOJcRTfEDACJQ6wPoRUWnR51QvwDCIJgt0ycYXRBTABIHAfupWjnLM6jDogUIAzDpariEqLb7XrsVx0XKcF13V987/fY0bIo4MA+EX5q4nv+RNwrz9u2ncD+YL/qmAKgaIzEG5mMw7gBoRkAEm+1WhcyVsWenAGVIiq5BsB0wHTAdMB0wHTAdMB0wHTAdCC9FatcKtvt9jHy92QN3JxnfutoDrF3Mg638XdlABQ5QvFDWTjHpv+Mc7mGvDXlM1Cv108B8SYLz2NTjuM4Z8oAjUbjBxD7VSAolnIoVxmAdlsFQopTrhaAeSGmxbUClEFkiWsHyIPIEy8EqNVq30gcUhA/8BO8X+znjXwmsP5MLx+4rPg/qL3E8KLnDDoAAAAASUVORK5CYII=" style="display: inline;" height="25" alt="xDai Bridge"/></p>
+	<p><input type="number" id="amount" /><img
```
Comment (James-Sangalli) [inline examples/action/xdai-bridge.en.shtml]: Cannot be number, don't support HTML5

Comment (colourful-land) [reply examples/action/xdai-bridge.en.shtml]: Use react to amend this?

File: examples/action/xdai-bridge.en.shtml (hunk 1 above)
Comment (JamesSmartCell) [reply examples/action/xdai-bridge.en.shtml]: ok James.

File: examples/action/xdai-bridge.en.shtml (hunk 2 above)
Comment (James-Sangalli) [reply examples/action/xdai-bridge.en.shtml]: Yes, another PR

**Pull request merged**